packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -102,7 +132,7 @@ export default class QuanticSearchBox extends LightningElement {
     this.resetHighlighted();
     suggestion.setAttribute('aria-selected', 'true');
     suggestion.classList.add('slds-has-focus');
-    this.input.value = suggestion.textContent;
+    this.input.value = suggestion.innerText;
```

Comment:
This is related to the rich text change above, is it because we want to be able to include html for highlighting?

Reply:
It's because with `textContent` there was a bug where if the user navigates through the suggestions with the keyboard, the input in the search box would not update and when the user pressed enter, it would submit a search with an empty string query. Using `innerText` fixed that.